repository/schema-registry/operator/templates/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: schema-registry
+spec:
+  replicas: 1
```

Comment:
let's parametrize this

Reply:
Done. Added parameters to configure this.